product_docs/docs/pgd/5/terminology.mdx

```diff
@@ -115,7 +115,8 @@ A PGD cluster is based around bidirectional replication, but in some use cases s
 
 #### Two-phase commit (2PC)
 
-A multi-step process for achieving consistency across multiple database nodes.
+A multi-step process for achieving consistency across multiple database nodes. A first phase sees a transaction prepared on an originating node and sent to all participating nodes. Each participating node validates that it can apply the transaction and 
```

Comment:
```suggestion
A multi-step process for achieving consistency across multiple database nodes. The first phase sees a transaction prepared on an originating node and sent to all participating nodes. Each participating node validates that it can apply the transaction and 
```